stock_view_adj/security/stock_inventory_security.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Allow Inventory/User to see the Inventory/Report menu. -->
+    <record model="ir.ui.menu" id="stock.menu_warehouse_report">
+        <field name="groups_id" eval="[(5, 0, 0)]"/> 
```

Comment:
Just FYI, you can find lines being `<field name="groups_id" eval="[(5,)]"/>` in Odoo's code.
https://github.com/odoo/odoo/blob/b3c345cf28ac0d02252d41cf613c6a296fa54b38/odoo/addons/base/data/res_users_demo.xml#L83
Your code should also do. 👍

Reply:
As far as the Odoo referencing guidelines, it is to be ignored if no argument is specified. Usually, it seems to specify 0 or False.

https://www.odoo.com/documentation/12.0/reference/orm.html#set-operations
> Values marked as _ in the list above are ignored and can be anything, generally 0 or False.

Both of the following writes seem to exist in Odoo's code. I'll merge as is this time.  
Thanks 😃 

* `eval="[(5, 0, 0)]"`
* `eval="[(5,)]"`